test2.ics

```diff
@@ -1,4 +1,3 @@
-BEGIN:VCALENDAR
 VERSION:2.0
```

Comment:
<p><b style="color:black" >Rule</b>: Some Text<br>
    <input type="hidden" id="custId" name="custId" value="3487">
    <b style="color:black" >Severity</b>:&nbsp;<img src="https://cidersecurity-public-icons.s3.amazonaws.com/prSeverityIcons/high.svg" alt="HIGH" width="62px" align="center"><br>
    <b style="color:black" >Scanner:</b> <a href="https://medium.com/cider-sec/malicious-code-analysis-13d5c1b37ffe">Scanner Link</a></p>
<details>
    <summary><b style="color:black">More Info</b></summary>
    Some description text <br><br><a href="https://medium.com/cider-sec/malicious-code-analysis-13d5c1b37ffe">Read more about this issue</a>
</details>